[bug] Fix unit tests

In GitHub Actions (CI) or locally via the `make test`, both doing `go test ./... -cover -race`, we currently have this errors that needs to be fixed:
```none
--- FAIL: TestMassive (0.02s)
    workloads_test.go:136: 
        	Error Trace:	/home/runner/work/score-k8s/score-k8s/internal/convert/workloads_test.go:136
        	Error:      	Not equal:
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1,8 @@
        	            	+apiVersion: v1
        	            	+binaryData:
        	            	+  file: aGVsbG8gJHttZXRhZGF0YS5uYW1lfSB3b3JsZA==
        	            	+kind: ConfigMap
        	            	+metadata:
        	            	+  name: example-c1-file-d0e9aff012
        	            	+---
        	            	 apiVersion: v1
        	            	@@ -6,9 +13,2 @@
        	            	   name: example-c1-file-7a1ae64977
        	            	----
        	            	-apiVersion: v1
        	            	-binaryData:
        	            	-  file: aGVsbG8gJHttZXRhZGF0YS5uYW1lfSB3b3JsZA==
        	            	-kind: ConfigMap
        	            	-metadata:
        	            	-  name: example-c1-file-d0e9aff012
        	            	 ---
        	            	@@ -112,4 +112,4 @@
        	            	               - key: file
        	            	-                path: root.md
        	            	-              name: example-c1-file-7a1ae64977
        	            	+                path: binary
        	            	+              name: example-c1-file-d0e9aff012
        	            	           - configMap:
        	            	@@ -117,4 +117,4 @@
        	            	               - key: file
        	            	-                path: binary
        	            	-              name: example-c1-file-d0e9aff012
        	            	+                path: root.md
        	            	+              name: example-c1-file-7a1ae64977
        	            	 status: {}
        	Test:       	TestMassive
FAIL
```

Anyone would like to take this one?

While here, in Codespace, via the DevContainer here, we also have this error that could be fixed in this setup of the DevContainer, could be part of the fix in addition to the unit tests failure above or not:
```none
go: no such tool "covdata"
```

--- a/internal/convert/workloads.go
+++ b/internal/convert/workloads.go
@@ -16,6 +16,7 @@ package convert
 
 import (
 	"fmt"
+	"maps"
 	"slices"
 	"strings"
 
@@ -110,7 +111,8 @@ func ConvertWorkload(state *project.State, workloadName string) ([]machineryMeta
 			}
 			return sf(ref)
 		}
-		for target, volume := range container.Volumes {
+		for _, target := range slices.Sorted(maps.Keys(container.Volumes)) {
+			volume := container.Volumes[target]
 			if mount, vol, claim, err := convertContainerVolume(target, volume, state.Resources, volSubstitutionFunction); err != nil {
 				return nil, errors.Wrapf(err, "containers.%s.volumes.%s: failed to convert", containerName, target)
 			} else {
@@ -126,7 +128,8 @@ func ConvertWorkload(state *project.State, workloadName string) ([]machineryMeta
 			}
 		}
 
-		for target, f := range container.Files {
+		for _, target := range slices.Sorted(maps.Keys(container.Files)) {
+			f := container.Files[target]
 			if mount, cfg, vol, err := convertContainerFile(target, f, fmt.Sprintf("%s-%s-", workloadName, containerName), state.Workloads[workloadName].File, sf); err != nil {
 				return nil, errors.Wrapf(err, "containers.%s.files.%s: failed to convert", containerName, target)
 			} else {

--- a/internal/convert/workloads_test.go
+++ b/internal/convert/workloads_test.go
@@ -135,17 +135,17 @@ func TestMassive(t *testing.T) {
 	}
 	assert.Equal(t, `apiVersion: v1
 binaryData:
-  file: bXktY29udGVudCBleGFtcGxl
-kind: ConfigMap
-metadata:
-  name: example-c1-file-7a1ae64977
----
-apiVersion: v1
-binaryData:
   file: aGVsbG8gJHttZXRhZGF0YS5uYW1lfSB3b3JsZA==
 kind: ConfigMap
 metadata:
   name: example-c1-file-d0e9aff012
+---
+apiVersion: v1
+binaryData:
+  file: bXktY29udGVudCBleGFtcGxl
+kind: ConfigMap
+metadata:
+  name: example-c1-file-7a1ae64977
 ---
 apiVersion: v1
 kind: Service
@@ -245,13 +245,13 @@ spec:
           - configMap:
               items:
               - key: file
-                path: root.md
-              name: example-c1-file-7a1ae64977
+                path: binary
+              name: example-c1-file-d0e9aff012
           - configMap:
               items:
               - key: file
-                path: binary
-              name: example-c1-file-d0e9aff012
+                path: root.md
+              name: example-c1-file-7a1ae64977
 status: {}
 ---
 `, out.String())

